src/extension/netcdf-out/src/test/java/org/geoserver/wcs2_0/WCSNetCDFBaseTest.java

```diff
@@ -27,6 +27,7 @@ protected void setUpTestData(SystemTestData testData) throws Exception {
             "user_projections/" + netcdfProjectionsDefinition);
         final File projectionFile = new File(projectionFileDir, netcdfProjectionsDefinition);
         System.setProperty(NetCDFCRSAuthorityFactory.SYSTEM_DEFAULT_USER_PROJ_FILE, projectionFile.getCanonicalPath());
+        testData.copyTo(getClass().getResourceAsStream("reduced-cf-standard-name-table.xml"), "cf-standard-name-table.xml");
```

Comment:
Setup for this resource pulled up into this superclass to avoid an ugly stack trace in the test output when the table does no exist.